### DISCUSSION TITLE
WarpSpeed() Setup Help

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#109] [Help / Questions] [answered]
When using warp speed, is it possible to modify the size/material of the ground created?

**Comment by N8python** [comment]
Just found this in a code example. Would this be the correct solution?
```js
const { ground } = await this.third.warpSpeed()
```

**Comment by yandeu** [comment, marked as answer]
You could modify the ground using:
```js
const { ground } = await this.warpSpeed()
```
But, it is easier to just create a new one:

```js
this.warpSpeed('-ground')
const ground = this.physics.add.ground({ width: 10, height: 10 })
```

Also, take a look at the source code of the ground in [warpSpeed.ts](https://github.com/enable3d/enable3d/blob/master/packages/threeGraphics/src/plugins/warpSpeed.ts).
_The grid is a simple Base64 image string._

**Reply by N8python** [reply]
Ah I see. Thanks!
